### PR TITLE
PDS 2.0 testcase refactor - ScaleDownScaleupPXCluster

### PR DESCRIPTION
**Pull request by atrivedi-px**
**What this PR does / why we need it**:
PDS 2.0 testcase refactor - ScaleDownScaleupPXCluster

**Which issue(s) this PR fixes** (optional)
Closes #DS-7995

**Special notes for your reviewer**:

### CODE REVIEW

File: tests/unifiedPlatform/pds2/pds_dataservice_test.go [hunk 1]
```diff
@@ -355,3 +359,99 @@ var _ = Describe("{ValidatePdsHealthIncaseofFailures}", func() {
...
+				log.Debugf("Source Deployment Id: [%s]", *deployment.Create.Meta.Uid)
+				nodes, err := pdslib.GetNodesOfSS(*deployment.Create.Status.CustomResourceName, PDS_DEFAULT_NAMESPACE)
+				log.FailOnError(err, "Error while getting Data Serice Nodes")
+				nodeList = append(nodeList, nodes[0])
+
+			})
+
+			Step("Scale down PX Nodes on which Data Services are running", func() {
+				// disable PX Pod on the first node of each deployed Data Service
+				for _, nodeToStop := range nodeList {
```
Comment (Madanagopal19) [inline tests/unifiedPlatform/pds2/pds_dataservice_test.go:392]: can we write a function to stop px on a node ?

File: tests/unifiedPlatform/pds2/pds_dataservice_test.go [hunk 2]
```diff
@@ -355,3 +359,99 @@ var _ = Describe("{ValidatePdsHealthIncaseofFailures}", func() {
...
+			})
+
+			stepLog := "Running Workloads after scale down of PX Nodes"
+			Step(stepLog, func() {
+				_, err := WorkflowDataService.RunDataServiceWorkloads(*deployment.Create.Meta.Uid)
+				log.FailOnError(err, "Error while running workloads on ds")
+			})
+
+			Step("Scale Up PX  Nodes", func() {
+				for _, nodeToStart := range nodeList {
```
Comment (Madanagopal19) [inline tests/unifiedPlatform/pds2/pds_dataservice_test.go:421]: same like above, a method to start the px on the node.

**Review by Madanagopal19 — CHANGES_REQUESTED** [review]

**Review by Madanagopal19 — APPROVED** [review]

File: tests/unifiedPlatform/pds2/pds_dataservice_test.go (hunk 1 above)
Comment (atrivedi-px) [reply tests/unifiedPlatform/pds2/pds_dataservice_test.go]: Done

File: tests/unifiedPlatform/pds2/pds_dataservice_test.go (hunk 2 above)
Comment (atrivedi-px) [reply tests/unifiedPlatform/pds2/pds_dataservice_test.go]: Done

**Pull request merged**